ui-components/src/blocks/AppStatusItem.tsx

```diff
@@ -1,40 +1,39 @@
 import React from "react"
 import "./AppStatusItem.scss"
-import moment from "moment"
-import { Application, Listing } from "@bloom-housing/backend-core/types"
 import { LocalizedLink } from "../actions/LocalizedLink"
 import { t } from "../helpers/translator"
 
 interface AppStatusItemProps {
-  application: Application
-  listing: Listing
+  applicationDueDate?: string
+  applicationURL: string
+  applicationUpdatedAt: string
+  confirmationNumber?: string
+  listingName: string
+  listingURL: string
 }
 
 const AppStatusItem = (props: AppStatusItemProps) => {
```

Comment:
I wonder if we should rename this and the props as well to something even more generic like just `StatusItem`. That's what the CSS class names are already.